src/components/layouts/php-collection-page-layout.tsx

```diff
@@ -41,13 +41,24 @@ type CoursePageLayoutProps = {
   ogImageUrl: string
 }
 
+type ModuleResource = {
+  type: 'module'
+  resourceList: ResourceList
+  title: string
+  description: string
+}
+
+type NestedResource = ModuleResource | LessonResource
```

Comment:
Made a union type here so that the individually mapped resources can be typed properly